changelogs/head.asciidoc

```diff
@@ -9,6 +9,8 @@ https://github.com/elastic/apm-server/compare/9.0\...main[View commits]
 [float]
 ==== Bug fixes
 
+- Change permissions of `apm-server.yml` in `tar.gz` artifacts to `0600` {pull}15627[15627]
```

Comment:
Changelogs are a bit tricky now that we have so many active versions in dev. I _think_ we should add this line to both 8.18.asciidoc and 9.0.asciidoc, instead of head.asciidoc, as this change should be first released in 8.18.0 and 9.0.0 at the same time.

Reply:
See [`9650713` (#15627)](https://github.com/elastic/apm-server/pull/15627/commits/965071385b662b808aa78926c22974e383121762)